ngraph/frontend/paddlepaddle/src/model.cpp

```diff
@@ -378,6 +447,7 @@ namespace ngraph
         void InputModelPDPD::InputModelPDPDImpl::overrideAllOutputs(
             const std::vector<Place::Ptr>& outputs)
         {
+            m_graph_changed = true;
```

Comment:
Minor: if I comment this out - existing tests will be passed. Do we need new test verifying need of this flag on this line?

Reply:
New behavior shouldn't change existing tests, to really see the difference we need to add a partially unsupported model to tests.